Example/CustomActionControllers/Spotify.swift

```diff
@@ -165,9 +160,17 @@ open class SpotifyActionController: ActionController<SpotifyCell, ActionData, Sp
         super.init(nibName: nibNameOrNil, bundle: nibBundleOrNil)
         settings.behavior.bounces = true
         settings.behavior.scrollEnabled = true
-        settings.cancelView.showCancel = true
         settings.animation.scale = nil
         settings.animation.present.springVelocity = 0.0
+        settings.cancelView.hasCancelView = true
+        settings.cancelView.displayInFooter = false
+        
+        cancelView = Bundle.main.loadNibNamed("CancelView", owner: self, options: nil)?.first as? UIView
```

Comment:
I would go on the way of having a `cancelViewSpec`, similarly to the other view spec that we already have (`cellSpec`, `headerSpec`). I think that it shouldn't change too much how this will be used.

What do you think?